### PR TITLE
Bump Rubocop version and update rules accordingly

### PR DESCRIPTION
**Pull request by msaffitz**

### CODE REVIEW

File: lib/scold/hound.yml [hunk 1]
```diff
@@ -126,13 +127,23 @@ Style/TrailingCommaInArguments:
...
   Enabled: false
   EnforcedStyleForMultiline: no_comma
   SupportedStylesForMultiline:
   - comma
+  - consistent_comma
+  - no_comma
+Style/TrailingCommaInHashLiteral:
+  Description: Checks for trailing comma in hash literals.
+  StyleGuide: https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas
+  Enabled: false
```
Comment (stim371) [inline lib/scold/hound.yml:142]: Isn't this disabling the check?

I thought the rails team was operating with a "no trailing comma" rule. The frontend guild expects trailing commas everywhere.

File: scold.gemspec [hunk 2]
```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |s|
   s.require_paths = %w(lib)
   s.extra_rdoc_files = %w(README.md)
   s.rubygems_version = "2.5.0"
-  s.add_runtime_dependency("rubocop", "~> 0.37")
+  s.add_runtime_dependency("rubocop", "~> 0.54")
```
Comment (stim371) [inline scold.gemspec:28]: hound is at .51  https://github.com/houndci/linters/blob/master/Gemfile.lock#L79

also, changing this value doesn't really do much, projects that use the scold gem can install any rubocop gem up to but not including 1.0.

File: lib/scold/hound.yml (hunk 1 above)
Comment (msaffitz) [reply lib/scold/hound.yml:142]: It is, but that's the existing behavior.

File: scold.gemspec (hunk 2 above)
Comment (msaffitz) [reply scold.gemspec:28]: Yeah, I just figured I'd bump it up to what was explicitly tested.

**Review by stim371 — APPROVED** [review]

**Pull request merged**